src/react-leaflet-markercluster.js

```diff
@@ -60,7 +56,7 @@ export default class MarkerClusterGroup extends LayerGroup {
   componentWillReceiveProps(nextProps) {
     if (!isArraysEqual(this.props.markers, nextProps.markers)) {
       // Remove layer from map with previously rendered clustered markers
-      this.layerContainer.removeLayer(this.leafletElement);
+      // this.layerContainer.removeLayer(this.leafletElement);
```

Comment:
Was this change intentional?

Reply:
yep.

Because of removing `markers leaflet element` from `layerContainer` during component update `markers` was not added to the map, so you are adding/deleting markers from `Map` and instead of changes, all markers were removed.

Reply:
:+1:

Maybe just get rid of the line altogether and the preceding comment then?